tools/asr_evaluator/utils.py

```diff
@@ -119,7 +119,7 @@ def run_chunked_inference(cfg: DictConfig) -> DictConfig:
 
     # ctc model
     elif (cfg.pretrained_name and 'ctc' in cfg.pretrained_name.lower()) or (
-        cfg.pretrained_name and 'ctc' in cfg.pretrained_name.lower()
+        cfg.model_path and 'ctc' in cfg.model_path.lower()
     ):
         if cfg.inference.decoder_type and cfg.inference.decoder_type != 'ctc':
             raise ValueError(
```

Comment:
should it be "change 'decoder_type' in yaml to 'null' for CTC models"? For CTC/RNNT models, setting `decoder_type` will cause errors, since their `change_decoding_strategy()` function doesn't have the `decoder_type` param, only the hybrid model has it, right?